api/CSSFontFaceRule.json

```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
+    "CSSFontFaceRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSFontFaceRule",
```

Comment:
The current spec for this is https://drafts.csswg.org/css-fonts/#om-fontface, and it matches this, just the interface with a `style` property.